internal/models/cluster.go

```diff
@@ -75,6 +75,9 @@ type Cluster struct {
 
 	// CertificateAuthorityData for the cluster, encrypted at rest
 	CertificateAuthorityData []byte `json:"certificate-authority-data,omitempty"`
+
+	// MonitorHelmReleases to trim down the number of revisions per release
+	MonitorHelmReleases bool `json:"monitor_helm_releases"`
```

Comment:
```suggestion
	MonitorHelmReleases bool
```

JSON tag isn't necessary since this never gets used as an API type (other JSON tags are outdated)